packages/zipkin-instrumentation-request-promise/package.json

```diff
@@ -16,16 +16,16 @@
     "@types/request": "^2.0.8"
   },
   "dependencies": {
+    "asap": "^2.0.6",
+    "request": "^2.88.2",
+    "request-promise": "^4.2.5",
```

Comment:
I am not sure about this one. It feels like CLS is not supported in v4: https://github.com/request/request-promise#migration-from-v3-to-v4

Do you have any idea?

Reply:
I moved it from devDeps v 4.2.2 to deps 4.2.5 
where is v3?

Reply:
Yeah, right. Ignore this one.